doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3756,7 +3756,7 @@ NCSDK-15909: TF-M failing to build with Zephyr SDK 0.14.2
   * Increase the TF-M partition :kconfig:option:`CONFIG_PM_PARTITION_SIZE_TFM`.
   * Use Zephyr SDK version 0.14.1.
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
```suggestion
.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

NCS 2.5.0 uses an unaffected version of the "Zephyr SDK".

Reply:
Updated

Reply:
The reasoning in this feedback is wrong. I looked at the wrong text.

But I was accidentally correct.

So we should keep this proposed change.

We are not affected by NCSDK-16916.